### ISSUE TITLE
panic: runtime error: index out of range

### ISSUE THREAD

**Issue by programsurf** [Demonhero0/rlf#2]
I have a question. rlf generates panic error. I use Docker. 
Do you know the reason of error?

contract: ``0x07419940b9a2d1eb54e123bd4853240ffac77186``

command
``python3 -m rlf --proj /tmp/0x07419940b9a2d1eb54e123bd4853240ffac77186 --contract PowhCoin4 --fuzzer reinforcement --limit 2000 --reward cov+bugs --mode test --output_path /rlf/output/test``

log
```

root@ssb-System-Product-Name:/rlf# python3 -m rlf --proj /tmp/0x07419940b9a2d1eb54e123bd4853240ffac77186 --contract PowhCoin4 --fuzzer reinforcement --limit 2000 --reward cov+bugs --mode test --output_path /rlf/output/test
/usr/local/lib/python3.8/dist-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
cpu
Namespace(address=None, bug_rate=0.5, contract='PowhCoin4', dataset_dump_path=None, detect_bugs=None, execution='./execution.so', fuzzer='reinforcement', limit=2000, limit_time=1800, log_to_file=None, max_episode=50, mode='test', model='model_imitation', output_path='/rlf/output/test', proj='/tmp/0x07419940b9a2d1eb54e123bd4853240ffac77186', reward='cov+bugs', rl_model='model_dqn', seed=1, train_dir=None, v=1)
[2025-01-20 10:35:23,011][1737340523.011327][__main__][INFO] fuzzing start
{'PowhCoin4': {0: ['default'], 1: ['balanceOf', 'dividends', 'getMeOutOfHere', 'sellMyTokensDaddy', 'transfer', 'transferFrom', 'withdraw', 'withdrawOld'], 2: ['fund'], 3: ['approve', 'buyPrice', 'getEtherForTokens', 'getTokensForEther', 'reserve', 'sellPrice'], 4: []}}
load from cpu
load model from model_dqn/eval_net.pth
====================================
model DQN has been loaded...
====================================
/usr/local/lib/python3.8/dist-packages/sklearn/utils/deprecation.py:144: FutureWarning: The sklearn.preprocessing.data module is  deprecated in version 0.22 and will be removed in version 0.24. The corresponding classes / functions should instead be imported from sklearn.preprocessing. Anything that cannot be imported from sklearn.preprocessing is now part of the private API.
  warnings.warn(message, FutureWarning)
/usr/local/lib/python3.8/dist-packages/sklearn/base.py:313: UserWarning: Trying to unpickle estimator StandardScaler from version 0.20.2 when using version 0.22.1. This might lead to breaking code or invalid results. Use at your own risk.
  warnings.warn(
fuzz_loop_RL
[2025-01-20 10:35:23,397][1737340523.397153][rlf.fuzzers.environment][INFO] {"tx_count": 0, "num_contracts": 1, "insn_coverage": 0.0, "block_coverage": 0.0, "PowhCoin4": {"tx_count": 0, "covered_insns": 0, "insn_coverage": 0.0, "temp_insn_coverage": 0.0, "all_insns": 2981, "covered_blocks": 0, "block_coverage": 0.0, "temp_block_coverage": 0.0, "all_blocks": 261, "bugs": {}}}
[2025-01-20 10:35:23,397][1737340523.397340][rlf.fuzzers.environment][INFO] initial calls start
[2025-01-20 10:35:23,428][1737340523.428688][rlf.fuzzers.environment][INFO] initial calls end
[2025-01-20 10:35:23,428][1737340523.428825][rlf.fuzzers.environment][INFO] start reinforcement policy
panic: runtime error: index out of range

goroutine 17 [running, locked to thread]:
main._Cfunc_CString(0xc50b1d0000, 0x52c2f1b2, 0x52c2f1b2)
        _cgo_gotypes.go:47 +0x98
main.CommitTx(0x7644eb2e5410, 0x7644e8dcf110)
        /root/go/src/rlf/export/execution.go:68 +0x135
main._cgoexpwrap_eb50e75fb9e1_CommitTx(0x7644eb2e5410, 0x0)
        _cgo_gotypes.go:132 +0x5c
Aborted (core dumped)
```
